coriolis_openstack_utils/instances.py

```diff
@@ -120,3 +123,99 @@ def validate_migration_options(
         LOG.info("%s volume types are not mapped." %
                  instance_volume_types -
                  source_mapped_volume_types)
+
+
+def instances_assess(source_client, instances_names):
```

Comment:
Vague/unrelated function name.
`def get_instances_assessment`

Reply:
Also, split this into a function which only takes one instance name as an argument; and then just loop through all the names and call it here.